mexican-standoff.json

```diff
@@ -0,0 +1,287 @@
+{
+  "player_count": 3,
+  "labeling": [
+    [],
+    [
+      "alive3"
+    ],
+    [
+      "alive2"
+    ],
+    [
+      "alive2",
+      "alive3"
+    ],
+    [
+      "alive1"
+    ],
+    [
+      "alive1",
+      "alive3"
+    ],
+    [
+      "alive1",
+      "alive2"
+    ],
+    [
+      "alive1",
+      "alive2",
+      "alive3"
+    ]
+  ],
+  "transitions": {
+    "7": {
+      "0": {
+        "0": {
+          "0": [
+            0
+          ],
+          "1": [
+            2
+          ],
+          "2": [
+            2
+          ]
+        },
+        "1": {
+          "0": [
+            4
+          ],
+          "1": [
+            6
+          ],
+          "2": [
+            2
+          ]
+        },
+        "2": {
+          "0": [
+            4
+          ],
+          "1": [
+            6
+          ],
+          "2": [
+            2
+          ]
+        }
+      },
+      "1": {
+        "0": {
+          "0": [
+            1
+          ],
+          "1": [
+            3
+          ],
+          "2": [
+            3
+          ]
+        },
+        "1": {
+          "0": [
+            5
+          ],
+          "1": [
+            7
+          ],
+          "2": [
+            3
+          ]
+        },
+        "2": {
+          "0": [
+            4
+          ],
+          "1": [
+            6
+          ],
+          "2": [
+            2
+          ]
+        }
+      },
+      "2": {
+        "0": {
+          "0": [
+            1
+          ],
+          "1": [
+            1
+          ],
+          "2": [
+            1
+          ]
+        },
+        "1": {
+          "0": [
+            5
+          ],
+          "1": [
+            5
+          ],
+          "2": [
+            1
+          ]
+        },
+        "2": {
+          "0": [
+            4
+          ],
+          "1": [
+            4
+          ],
+          "2": [
+            0
+          ]
+        }
+      }
+    },
+    "0": {
+      "1": {
+        "1": {
+          "1": [
+            0
+          ]
+        }
+      }
+    },
+    "2": {
+      "1": {
+        "1": {
+          "1": [
+            2
+          ]
+        }
+      }
+    },
+    "4": {
+      "1": {
+        "1": {
+          "1": [
+            4
+          ]
+        }
+      }
+    },
+    "6": {
+      "1": {
+        "1": {
+          "1": [
+            6
+          ]
+        },
+        "2": {
+          "1": [
+            6
+          ]
+        }
+      },
+      "2": {
+        "1": {
+          "1": [
+            4
+          ]
+        },
+        "2": {
+          "1": [
+            4
+          ]
+        }
+      }
+    },
+    "1": {
+      "1": {
+        "1": {
+          "1": [
+            1
+          ]
+        }
+      }
```

Comment:
Here is an an error. In this case no player can choose 0, but all of them can choose 1.

(This issue has already been discussion in person)

Reply:
I figured out the bug, fixed in ce9f810.
Resulting json:
```
    "5": {
      "0": {
        "0": {
          "0": [
            5
          ],
          "1": [
            1
          ]
        }
      },
      "1": {
        "0": {
          "0": [
            4
          ],
          "1": [
            0
          ]
        }
      }
    }
  },
```
In state 5, if wait, still state 5. If only p3 kills, then state 1, if only p1 kills, then state 4, if both kill, then state 0.